### PR TITLE
[Data object versions] Show version note

### PR DESCRIPTION
**Pull request by BlackbitNeueMedien**
In some use-cases it is good to add a description or some meta data to data object versions. There is a field `note` in the `versions` table. Currently this can only be filled by manually calling the `Version::setNote()` method as far as I know. But currently this field is not displayed anywhere in the Pimcore backend. With this PR the version note gets shown in the data object preview / comparison - only if it is filled. So for projects which do not use version notes everything will look the same as before but for those which do the version note is displayed.

### CODE REVIEW

**Comment by dvesh3** [comment]
@BlackbitNeueMedien you also pass optional parameter on save like `$object->save(["versionNote" => "my new version"]);` and it displayed on versions list next to scheduled column:
image

and it would be nice to see it in version preview as well. thanks for your PR! 😊

File: bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/previewVersion.html.twig [hunk 1]
```diff
@@ -16,6 +16,14 @@
         <th>Key</th>
         <th>Value</th>
     </tr>
+    {% if versionNote %}
+        <tr class="system">
+            <td>Version Note</td>
+            <td>&nbsp;</td>
+            <td>{{ versionNote }}</td>
+        </tr>
+        <p><b>Version Note: </b> {versionNote}</p>
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/previewVersion.html.twig:25]: having a row is enough...please kick it 
image and please move it under "published". thanks!

**Comment by BlackbitNeueMedien** [comment]
Oh I see - my screen was only too small ;-)
Bildschirmfoto 2021-03-11 um 09 28 12

But nevertheless makes sense to also display it in version preview.

File: bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/previewVersion.html.twig (hunk 1 above)
Comment (BlackbitNeueMedien) [reply bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/previewVersion.html.twig]: Fixed. The `<p>` was actually not supposed to be committed.

**Comment by BlackbitNeueMedien** [comment]
Do you really think that it makes more sense below `Published` field? I thought that it would mix up contexts because id, key etc. (system fields) belong to the object while not note belongs to the version. For this reason I put it on top. But now I have put it under `Published` as you requested. What do you think?

**Comment by dvesh3** [comment]
> Do you really think that it makes more sense below `Published` field? I thought that it would mix up contexts because id, key etc. (system fields) belong to the object while not note belongs to the version. For this reason I put it on top. But now I have put it under `Published` as you requested. What do you think?

I would like to see this at the last position to be consistent with list view and the context is already mixed since date belongs to version, which is on top.

**Comment by dvesh3** [comment]
@BlackbitNeueMedien thanks a lot!

**Pull request merged**